### ISSUE TITLE
USER STORY: Image Augmentation

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/breast-cancer-detection#11]
As a **Developer** I can **augment the images** so that **the model's prediction accuracy is increased and overfitting decreased**.

**Acceptance Criteria**
* Use ImageDataGenerator to maximise the number of images in the training dataset by flipping, shifting weight and height, rescaling and zooming. 
* Use ImageDataGenerator to increase the number of images in the validation and test sets by rescaling.
